test/e2e/scenarios/no_np.go

```diff
@@ -0,0 +1,19 @@
+// Test the scenario where there are no network policies
+// implemented in which case all pods should be able to
+// communicate with each other
+
+package no_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	// . "github.com/onsi/gomega"
+)
+
+var _ = PDescribe("No network policy is implemented", func() {
+	PIt("All pods should be able to communicate", func() {
+		// TODO: Implement below functions
```

Comment:
You may write:

PBy("creating a new namespace in cluster 2, with a random name"

PBy("creating a pod in cluster 1, default test namespace")
PBy("creating a pod in cluster 2, in the random name namespace")
PBy("testing connectivity between pods")

Without implementation details, that's something anyone can read and understand.

For this, (later on implementation) you can use the same as the e2e basic dataplane of submariner,
we may need to add some tweaks for the namespace stuff.

Reply:
I didn't find PBy defined anywhere. Not even at https://godoc.org/github.com/onsi/ginkgo.
I tried running it and got below error

```
$ go test -args -dp-context cluster2 -dp-context cluster3 -ginkgo.v
# github.com/submariner-io/coastguard/test/e2e/scenarios
scenarios/no_np.go:14:3: undefined: PBy
FAIL	github.com/submariner-io/coastguard/test/e2e [build failed]
```

Reply:
Using "By" instead. Thanks @mangelajo for the suggestion :)